pyproject.toml

```diff
@@ -71,11 +71,27 @@ testpaths = "assesspy/tests"
 legacy_tox_ini = """
 [tox]
 min_version = 4.0
-envlist = py39, py310, py311, py312, py313
+envlist =
+    py{39, 310}-lowest
```

Comment:
The `-lowest` set of tox environments here use the lowest possible version of direct dependencies (i.e. the min specified in `pyproject.toml`). The other envs just use the latest possible version of each package.

`pyarrow==9.0.0` isn't compatible with Python >= 3.11, otherwise these two sets of environments could overlap more.